src/urllib3/request.py

```diff
@@ -1,3 +1,4 @@
+import json as json_serializer
```

Comment:
Can we alias this as a private name (ie `_json`) so it doesn't appear to be a public member of `urllib3.request`?